normalizer.py

```diff
@@ -43,6 +44,13 @@ def craft(self, docs: DocumentArray, **kwargs) -> DocumentArray:
             doc.blob = img
         return filtered_docs
 
+    def read(self, docs):
+        for doc in docs:
+            if doc.uri:
+                doc.convert_image_uri_to_blob()
+            elif doc.buffer:
+                doc.convert_image_buffer_to_blob()
+
```

Comment:
```suggestion
    def _convert_image_to_blob(self, doc: Document):
        if doc.uri:
            doc.convert_image_uri_to_blob()
        elif doc.buffer:
            doc.convert_image_buffer_to_blob()

```

Reply:
I prefer to use `_convert_image_to_blob` as internal function, which is not explicit public api.

Reply:
Addressed, please see [cbdeeb4](https://github.com/jina-ai/executor-image-normalizer/pull/11/commits/cbdeeb4baba0007618176dc489dc6d59780ed9a7).